spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -1220,6 +1220,10 @@ In addition to the metadata fields, `@Counted` has the following field:
 `monotonic`:: If `false` (default), the counter is decremented when the annotated method returns, counting current invocations of the annotated method.
 If `true`, the counter increases monotonically, counting total invocations of the annotated method.
 
+WARNING: The normal expectation is that each invocation of of a method marked with `@Counted` would result in the
```

Comment:
One too many "of"s in "invocation of of a method"

I'm not sure about using the word "normal", maybe "intuitive" is better?

Reply:
Yes the latter would be more intuitive :)